modules/order/commerce_order.module

```diff
@@ -10,6 +10,17 @@ use Drupal\field\Entity\FieldConfig;
 use Drupal\field\Entity\FieldStorageConfig;
 
 /**
+ * Implements hook_theme().
+ */
+function commerce_order_theme($existing, $type, $theme, $path) {
+  return [
+    'order_edit_form' =>  [
```

Comment:
I'll do it anyways. I'm honestly not sure. But I remember someone opened a ticket about Layout plugin module not doing the theme namespacing.